adapters/gumgum/usersync_test.go

```diff
@@ -5,12 +5,13 @@ import (
 	"text/template"
 
 	"github.com/prebid/prebid-server/privacy"
+	"github.com/prebid/prebid-server/privacy/ccpa"
 	"github.com/prebid/prebid-server/privacy/gdpr"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestGumGumSyncer(t *testing.T) {
-	syncURL := "https://rtb.gumgum.com/usync/prbds2s?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&r=localhost%2Fsetuid%3Fbidder%3Dgumgum%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D"
+	syncURL := "https://rtb.gumgum.com/usync/prbds2s?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r=localhost%2Fsetuid%3Fbidder%3Dgumgum%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D"
 	syncURLTemplate := template.Must(
```

Comment:
Both `{{.GDPR}}` and `{{.GDPRConsent}}` are parsed both in the query and in the encoded URL part. Don't we also want to test for the `us_privacy` value to be included in the encoded URL?

Reply:
The `us_privacy` is not included in the callback url to prebid server because prebid server doesn't make use of that value. It would only really be needed right now if others are parsing the callback url and extracting those values.. but I don't think that's a valid use case.